src/docs/Product/approveProductRequest.yaml

```diff
@@ -0,0 +1,30 @@
+paths:
+  /api/product/{productId}/requesters/{requesterId}/approve:
+    patch:
+      tags:
+        - Product
+      summary: Updats beneficiary of the product (Requires log in)
+      description: Approves a product request , Sets benficiary to be the requesterID , and updates list of recieved product or requester
+      operationId: updateProduct
+      parameters:
+        - name: productId
+          in: path
+          required: true
+          schema:
+            type: string
+            example: 62f661cf728303c5bead3b6d
+        - name: requesterId
+          in: path
+          required: true
+          schema:
+            type: string
+            example: 62f661cf728303c5bead3b6d
```

Comment:
this example is a product id, you can change it with this one, (owner of the product id above): 62e2f022552347123617a73f

Reply:
Done